tests/Makefile.tests_common

```diff
@@ -1,7 +1,7 @@
-ifneq (,$(filter driver_%,$(APPLICATION)))
+APPLICATION ?= tests_$(shell basename $(CURDIR))
```

Comment:
I think you should be able to do `$(not directly $(CURDIR)) ` or some variant thereof to avoid calling basename

Reply:
Yes, nice! Changed.

(will have to also use that to where we set the default module name)